bugsnag-android-core/src/main/java/com/bugsnag/android/NativeInterface.java

```diff
@@ -38,6 +42,36 @@ private static Client getClient() {
         }
     }
 
+    /**
+     * Create an empty Event for a native "handled exception" report. The returned Event will have
+     * no Error objects, metadata, breadcrumbs, or feature flags. It's indended that the caller
+     * will populate the Error and then pass the Event object to
+     * {@link Client#populateAndNotifyAndroidEvent(Event, OnErrorCallback)}.
+     */
+    private static Event createNativeEvent() {
```

Comment:
What about this makes it inherently a native event? Does `createEmptyEvent()` make sense here (or perhaps as a convenience initializer on Event)?

Reply:
Actually I can use an existing constructor and get the same effect.